sdk/Lusid.Sdk.Tests/tutorials/Ibor/StructuredResultStore.cs

```diff
@@ -0,0 +1,946 @@
+            
+            // Create and upsert the recipe
+            var configurationRecipe = new ConfigurationRecipe(scope, portfolioCode, new MarketContext(), pricingContext);
+            var upsertRecipeRequest = new UpsertRecipeRequest(configurationRecipe, null);
+            _recipeApi.UpsertConfigurationRecipe(upsertRecipeRequest);
+            
+            
+            
+            // Create a valuation request requesting for:
+            // - Lusid instrument id
+            // - Accrual as is held in UnitResults (one from the document)
+            // - Pv that was valued based on the overriden accrual
+            // - The scaled accrual.
+            // - Client Custom value that gets carried on.
+            var valuationRequest = new ValuationRequest(
+                recipeId: new ResourceId(scope, portfolioCode),
+                metrics: new List<AggregateSpec>
+                {
+                    new AggregateSpec(TestDataUtilities.Luid, AggregateSpec.OpEnum.Value),
+                    new AggregateSpec("UnitResult/Accrual", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec(TestDataUtilities.ValuationPv, AggregateSpec.OpEnum.Value),
+                    new AggregateSpec("Holding/default/Accrual", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec("UnitResult/ClientCustomValue", AggregateSpec.OpEnum.Value),
+                },
+                portfolioEntityIds: new List<PortfolioEntityId> { new PortfolioEntityId(scope, portfolioCode) },
+                valuationSchedule: new ValuationSchedule(effectiveAt: effectiveAt));
+    
+            // Perform valuation and obtain results
+            var results = _apiFactory.Api<IAggregationApi>().GetValuation(valuationRequest);
+            
+            // We expect the following output
+            // | LusidInstrumentId | Accrual (UnitResult - Unscaled) | Pv Ammount | Accrual (Holding - scaled) | ClientCustomValue |
+            // | ----------------- | ------------------------------- | ---------- | -------------------------- | ----------------- |
+            // | <generated-luid>  | 0.123456                        | 107.23456  | 1.23456                    | 1.7320508         |
+            Assert.That(results.Data[0]["Instrument/default/LusidInstrumentId"], Is.EqualTo(luids.First()));
+            Assert.That(results.Data[0]["UnitResult/Accrual"], Is.EqualTo(0.0123456));
+            Assert.That(results.Data[0]["Valuation/PV/Amount"], Is.EqualTo(107.23456));
+            Assert.That(results.Data[0]["Holding/default/Accrual"], Is.EqualTo(1.23456));
+            Assert.That(results.Data[0]["UnitResult/ClientCustomValue"], Is.EqualTo(1.7320508));
+        }
+
+        // In this example grouped level structured result store example is shown. Showing how document containing multiple portfolios can be upserted and queried.
+        [Test]
```

Comment:
add `[LusidFeature]`

Reply:
Added the feature codes, and added them in Lusid-Features as well, if okay mind approving ?
https://gitlab.com/finbourne/salesengineering/lusid-features/-/merge_requests/39 
@rizwansaeed